libs/api-mocks/floating-ip.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+
+import type { FloatingIp } from '@oxide/api'
+
+import { instance, project } from '.'
+import type { Json } from './json-type'
```

Comment:
This will fix the confusing test failure.

```suggestion
import { instance } from './instance'
import type { Json } from './json-type'
import { project } from './project'
```

Vite does not like the import from `'.'`. I suspect it is due to the circular import: `./index.ts` imports this file, and we're importing from it. There are very few spots we import `from '.'`, and they should probably be changed. I'll do that separately and see if there's a lint rule we can turn on.

```
$ rg "from '.'"
libs/api/roles.ts
18:import { usePrefetchedApiQuery } from '.'

libs/util/math.spec.ts
10:import { GiB } from '.'

libs/util/math.ts
9:import { splitOnceBy } from '.'

libs/table/cells/BooleanCell.tsx
14:import type { Cell } from '.'

libs/table/cells/EnabledCell.tsx
11:import type { Cell } from '.'

libs/table/cells/TypeValueListCell.tsx
8:import { TypeValueCell, type Cell, type TypeValue } from '.'

libs/table/cells/FirewallFilterCell.tsx
11:import { TypeValueCell, type Cell } from '.'
```